src/autonlp/autonlp.py

```diff
@@ -103,6 +103,10 @@ def create_project(self, name: str, task: str, language: str, max_models: int):
 
     def create_evaluation(self, task: str, dataset: str, model: str, col_mapping: str, split: str, config: str = None):
```

Comment:
should `col_mapping` be optional if `task` is `text_classification` (because then the column mapping is handled by `datasets.prepare_for_task`)?

if we keep it as a required parameter, i'll update the condition we have in the backend to make sure we can reach the task templates if both `col_mapping` is provided and `task="text_classification"`: https://github.com/huggingface/autonlp-backend/pull/232/files#diff-d8b12048e47b2a5ebd119e3f114613ca482f8b6cd85cbba2ab4eaf663c066ce7R19

Reply:
if user specifies `datasets` "tasks", `col_mapping` can be optional. right?

Reply:
yes, that's correct